src/Symfony/Component/HttpKernel/Tests/Fixtures/Controller/AttributeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\Controller;
+
+use Symfony\Component\HttpKernel\Tests\Fixtures\Attribute\Foo;
+
+class AttributeController
+{
+    public function action(
+        #[Foo('bar')]
+        string $baz
+    ) {
```

Comment:
Can you move everything on one line? (same below)

Reply:
I have use this style so that the code was parsable on a lower PHP version but since it's a fixture file that should be read only on PHP 8 it's not really important.